org.eclipse.scanning.sequencer/src/org/eclipse/scanning/sequencer/watchdog/DeviceController.java

```diff
@@ -82,12 +82,13 @@ public void seek(String id, int stepNumber) throws ScanningException, Interrupte
 		// If any of the others think it should be paused, we do not resume
 		Map<String, DeviceState> copy = new HashMap<>(states);
 		copy.put(id, DeviceState.RUNNING);
+		if (device.getDeviceState()!=DeviceState.PAUSED) return; // Cannot seek it.

```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') At most one statement is allowed per line, but 2 statements were found on this line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS00122)